README.md

```diff
@@ -71,6 +71,7 @@ This is the running list of what in tech has been affected by COVID-19. Pull req
 | Glitch | Required | Restricted | Restricted | Restricted | 2020-03-09 |
 | Google[[1]](https://www.theguardian.com/world/2020/mar/04/coronavirus-google-tech-dublin-twitter-work-from-home#maincontent) [[2]](https://www.reuters.com/article/us-health-coronavirus-alphabet/google-recommends-all-north-america-employees-to-work-from-home-idUSKBN20X38J?feedType=RSS&feedName=technologyNews) | Encouraged / Geo-specific | Restricted | Restricted | ? | 2020-03-11 |
 | Groupon | Encouraged | Restricted | ? | Restricted | 2020-03-06 |
+| Heap | Required | Restricted | Restricted | Restricted | 2020-03-09 |
```

Comment:
```suggestion
| [Heap](https://github.com/phildini/stayinghomeclub/pull/345) | Required | Restricted | Restricted | Restricted | 2020-03-09 |
```

Since you're the CEO, do you mind just using this as a source?

Reply:
Lgtm!